README.md

```diff
@@ -6,6 +6,15 @@
     Product focuses on 100% education as well as upskilling developing countries and rural areas.</i></b>
 </blockquote>
 
+<br/>
+
+![Issues tag](https://img.shields.io/github/issues/codeforcauseorg/edu-client)
+![Forks tag](https://img.shields.io/github/forks/codeforcauseorg/edu-client)
+![Stars](https://img.shields.io/github/stars/codeforcauseorg/edu-client?style=social)
+![Licence](https://img.shields.io/github/license/codeforcauseorg/edu-client)
+![twitter](https://img.shields.io/twitter/follow/codeforcauseIn?style=social)
+![Languages](https://img.shields.io/github/languages/count/codeforcauseorg/edu-client)
+
```

Comment:
Align it to center and remove language shield

Reply:
Also add discord badge

Reply:
Ok

Reply:
For discord, the server admin has to enable widget settings on this server. Could you please do that